src/test/java/com/puppycrawl/tools/checkstyle/internal/powermock/PropertyCacheFilePowerTest.java

```diff
@@ -78,26 +70,6 @@ protected String getPackageLocation() {
         return "com/puppycrawl/tools/checkstyle/propertycachefile";
     }
 
-    @Test
-    public void testCloseAndFlushOutputStreamAfterCreatingHashCode() throws IOException {
```

Comment:
I am not sure on the best way to keep these tests since these are streams we aren't creating and the class doing the flush/close is implementation specific.

Reply:
I just removed the catch that wasn't being covered as it isn't really needed since the calling method also has a catch statement that caught the exception being thrown. There is nothing validating the message of the exception being thrown or anything, so it seems pointless.